### DISCUSSION TITLE
Add relevant links to docs site

### DISCUSSION THREAD

**Discussion by destructobeam** [alpinejs/alpine#3257] [7. Bugs (archived)]
On the [Alpine site](https://alpinejs.dev/) only the `/` route has a link to the GitHub project in the navigation.

This makes it difficult to find unless you remember the one page it’s on and I have ended up searching in a new tab more than once.

Would be nice to have this everywhere, or a links page in the navigation or links in the footer, with any social/video site accounts as well.

**EDIT:** Seems the links are only missing on the small(er/est) breakpoints.

**Comment by ekwoka** [comment]
<img width="1409" alt="Screenshot 2022-11-04 at 12 34 45 PM" src="https://user-images.githubusercontent.com/43540491/199928586-6d28e72c-2984-489c-a755-ac89f4ce59ea.png">

Looks like it's on every page to me..

**Reply by ekwoka** [reply]
looks like only mobile has it in only that one spot

**Reply by destructobeam** [reply]
Sorry, yeah, I have been using a breakpoint that hides it I guess, worth a fix though still IMO?

**Reply by ekwoka** [reply]
Yeah probably. The docs site is in a private repo, so we can hope 🤞
